### PR TITLE
WIP: fix docker build in makefile

### PR DESCRIPTION
**Pull request by zryfish**
Signed-off-by: Jeff <zw0948@gmail.com>

**What type of PR is this?**
/kind flake

**What this PR does / why we need it**:
Currently, target `docker-build-no-test` in makefile will build image and push to repository. This PR will remove push process when building image locally.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,9 +90,9 @@ openapi:
 	go run ./tools/cmd/doc-gen/main.go
 # Build the docker image
 docker-build: all
-	hack/docker_build.sh
+	hack/docker_build.sh ${TAG} ${REPO}
 docker-build-no-test: ks-apiserver ks-controller-manager
-	hack/docker_build.sh
+	hack/docker_build.sh ${TAG} ${REPO}
```
Comment (LinuxSuRen) [inline Makefile:95]: This is a breaking change. I'd suggest that adding a new instruction.

File: hack/docker_build.sh [hunk 2]
```diff
@@ -30,22 +30,14 @@ get_repo() {
...
+# push to $REPO with default $TAG tag
 TAG=$(tag_for_branch "$1")
 REPO=$(get_repo "$2")
 
-# Push image to dockerhub, need to support multiple push
-cat ~/.docker/config.json | grep index.docker.io
-if [[ $? != 0 ]]; then
-  echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-fi
-
```
Comment (LinuxSuRen) [inline hack/docker_build.sh:42]: Why delete these lines?

**Review by LinuxSuRen — COMMENTED** [review]
Please provide a more readable PR title. I don't know what issues do you want to fix.

**Comment by zryfish** [comment]
/close
Having some issues needed to be settled.

**Pull request closed without merging**